modules/osd-persistent-storage-csi-efs-sts.adoc

```diff
@@ -6,11 +6,11 @@
 
 :_content-type: PROCEDURE
 [id="efs-sts_{context}"]
-= Configuring AWS EFS CSI Driver Operator with Secure Token Service
+= Configuring Amazon EFS CSI Driver Operator with Secure Token Service
 
-This procedure explains how to configure the AWS EFS CSI Driver Operator with {product-title} on AWS Secure Token Service (STS).
+This procedure explains how to configure the Amazon EFS CSI Driver Operator with {product-title} on AWS Secure Token Service (STS).
```

Comment:
Revoked.

Reply:
Updated, "This procedure explains how to configure the [AWS EFS CSI Driver Operator](https://github.com/openshift/aws-efs-csi-driver-operator) (a Red Hat operator) with {product-title} on AWS Secure Token Service (STS)."